data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -416,6 +416,7 @@ then
         ret=$?
         set +u
         cluster_bootstrap_gather
+	check_url_resolution
```

Comment:
How is this function being referenced from the other file? I don't see the bootstrap-verify-dns-config.sh script being sourced.